installation.rst

```diff
@@ -45,6 +45,26 @@ your system and video driver. The OpenGL version must be at least 2.1.
 Installation options
 ====================
 
+**Before installing VisPy** you should ensure a working version of python is
+installed on your computer, including all of the requirements included in the
+**Backend Requirements** section above. A simple wayto install most of these
+requirements is to install the **Anaconda** scientific python distribution
+from Continuum Analytics.
+`Anaconda <https://www.anaconda.com/download/>`_ will
+install most of the VisPy dependencies for you. If your computer is low on hard
+disk space, or you would like a minimal python installation, you may install
+the `Miniconda <https://conda.io/miniconda.html>`_ package also from
+Continuum Analytics. Once Anaconda is installed, create a
+`conda python environment <https://conda.io/docs/user-guide/tasks/manage-python.html>`_.
+
+Next, install the following VisPy dependencies directly through `pip` or the Anaconda package installer.
+
+$ conda install numpy pyqt
```

Comment:
this can be:
```
.. code-block:: console

    $ conda install numpy pyqt
```
and it will render nicely

Reply:
(same with the `$ pip install` line below and possibly others)

Reply:
With or without the `$`?

Reply:
With, it renders it with highlighting like this

https://6100-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.2wP5EEF/html/install_mne_python.html